### ISSUE TITLE
Don't hardcode env var KINDLEGEN in build.sh

### ISSUE THREAD

**Issue by fahrinh** [mraible/infoq-mini-book#8]
It will give false assumption when building even though KINDLEGEN is correctly setup.

I got this error when building :
```
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- kindlegen
        at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:961)
        at RUBY.require(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:54)
        at RUBY.distill_epub_to_mobi(uri:classloader:/gems/asciidoctor-epub3-1.5.0.alpha.7/lib/asciidoctor-epub3/packager.rb:611)
        at RUBY.package(uri:classloader:/gems/asciidoctor-epub3-1.5.0.alpha.7/lib/asciidoctor-epub3/packager.rb:602)
        at RUBY.write(uri:classloader:/gems/asciidoctor-epub3-1.5.0.alpha.7/lib/asciidoctor-epub3/converter.rb:41)
        at RUBY.write(uri:classloader:/gems/asciidoctor-1.5.6.1/lib/asciidoctor/document.rb:1083)
        at RUBY.convert(uri:classloader:/gems/asciidoctor-1.5.6.1/lib/asciidoctor.rb:1507)
        at RUBY.block in convert_file(uri:classloader:/gems/asciidoctor-1.5.6.1/lib/asciidoctor.rb:1575)
        at org.jruby.RubyIO.open(org/jruby/RubyIO.java:1156)
        at RUBY.convert_file(uri:classloader:/gems/asciidoctor-1.5.6.1/lib/asciidoctor.rb:1575)
        at RUBY.convertFile(<script>:75)
```

**Comment by mraible** [comment]
This will be fixed when #1 is fixed.

**Comment by wimdeblauwe** [comment]
I fixed this in PR #9
